blocksuite/affine/data-view/src/__tests__/table-group.unit.spec.ts

```
@@ -0,0 +1,32 @@
+import { describe, test, expect } from 'vitest';
+
+import { TableGroup } from '../view-presets/table/pc/group.js';
+import { pcEffects } from '../view-presets/table/pc/effect.js';
+import { MobileTableGroup } from '../view-presets/table/mobile/group.js';
+import { mobileEffects } from '../view-presets/table/mobile/effect.js';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)